src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -79,6 +79,16 @@ public override string Name
         public abstract AssemblyIdentity Identity { get; }
 
         /// <summary>
+        /// Assembly version pattern with wildcards represented by <see cref="ushort.MaxValue"/>,
+        /// or null if the version string specified in the <see cref="AssemblyVersionAttribute"/> doesn't contain a wildcard.
```

Comment:
Just curious: why null if there are no wildcards rather than the actual version?